GameOfHive/MenuController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  MenuView.swift
+//  GameOfHive
+//
+//  Created by Tomas Harkema on 21-05-15.
+//  Copyright (c) 2015 Beetles. All rights reserved.
+//
+
+import UIKit
+
+enum MenuPressedState {
+  case Show
+  case HalfPressed
+  case Hide
+}
+
+class MenuController: UIViewController {
+
+    @IBOutlet weak var centerButton: HiveButton!
+
+    var menuState = false
+    var buttons = [HiveButton]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        addButtons()
```

Comment:
viewWillAppear kan meerdere keren aangeroepen worden toch? Dan moeten we niet meerdere keren buttons toevoegen